src/main.py

```diff
@@ -65,6 +71,7 @@ def setup(self):
         self.input_debug_hud = DebugHud(self.player_manager.players)
 
         # Player Huds
+        # WHEN WE CREATE HUD, EACH PLAYER SHOULD ALREADY HAVE GAME_MODE_STATE ON IT
```

Comment:
Should probably remove this comment?  Or at least make it lowercase.  Pretty sure I added this while we were brainstorming.